k8s_build_jobs.py

```diff
@@ -33,7 +33,10 @@ def main(jobs_dir: Path=JOBS_DIR):
     ])
 
     for dataset in datasets:
-        template = yaml.load(JOB_TEMPLATE.read_text())
+        template = yaml.load(
+            JOB_TEMPLATE.read_text(),
+            Loader=yaml.FullLoader  # type: ignore
```

Comment:
no idea wtf is going on with mypy complaining that `FullLoader` doesn't exist on `yaml` here